scripts/deploy/contracts/InvestorVesting.ts

```diff
@@ -20,6 +43,13 @@ export const deploy = async (deployer, setAddresses) => {
   )) as InvestorVesting;
   console.log(`deployed InvestorVesting to address ${investorVesting.address}`);
   setAddresses({ InvestorVesting: investorVesting.address });
+  console.log("Verifying InvestorVesting on Etherscan");
+  // Wait for a bit for etherscan to pick up the contract.
+  await sleep(5000);
```

Comment:
Why do you need to wait 5 seconds here?

Reply:
Before I added that it was trying to verify the contract before etherscan had indexed the transaction, so I just gave it some time to pick up the tx